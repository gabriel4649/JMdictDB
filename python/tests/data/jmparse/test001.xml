<!-- 
This file contains test data in jmdict XML format for driving
unit and higher-level tests.   
-->

<entry>
<!-- Minimal entry, hiragana -->
<ent_seq>3000010</ent_seq>
<r_ele>
<reb>あああ</reb>
</r_ele>
<sense>
<gloss>test10</gloss>
</sense>
</entry>
<entry>
<!-- Minimal entry, katakana -->
<ent_seq>3000020</ent_seq>
<r_ele>
<reb>テスト</reb>
</r_ele>
<sense>
<gloss>test20</gloss>
</sense>
</entry>
<entry>
<!-- Minimal entry, kanji -->
<ent_seq>3000030</ent_seq>
<k_ele>
<keb>発音</keb>
</k_ele>
<r_ele>
<reb>はつおん</reb>
</r_ele>
<sense>
<gloss>test30</gloss>
</sense>
</entry>
<entry>
<!-- Multiple reading -->
<ent_seq>3000040</ent_seq>
<r_ele>
<reb>はつおん</reb>
</r_ele>
<r_ele>
<reb>ばしょう</reb>
</r_ele>
<r_ele>
<reb>バショー</reb>
</r_ele>
<sense>
<gloss>test40</gloss>
</sense>
</entry>
<entry>
<!-- Multiple kanji -->
<ent_seq>3000050</ent_seq>
<k_ele>
<keb>発音</keb>
<keb>本屋</keb>
<keb>英会話</keb>
</k_ele>
<r_ele>
<reb>はつおん</reb>
</r_ele>
<sense>
<gloss>test50</gloss>
</sense>
</entry>
<entry>
<!-- Multiple senses -->
<ent_seq>3000060</ent_seq>
<r_ele>
<reb>あい</reb>
</r_ele>
<sense>
<gloss>test60-1</gloss>
</sense>
<sense>
<gloss>test60-2</gloss>
</sense>
<sense>
<gloss>test60-3</gloss>
</sense>
</entry>
<entry>
<!-- Multiple glosses -->
<ent_seq>3000070</ent_seq>
<r_ele>
<reb>あえ</reb>
</r_ele>
<sense>
<gloss>test70-1</gloss>
<gloss>test70-2</gloss>
<gloss>test70-3</gloss>
</sense>
</entry>

<entry>
<!-- embedded lit text -->
<ent_seq>3000080</ent_seq>
<r_ele>
<reb>あえ</reb>
</r_ele>
<sense>
<gloss>gloss text2 (lit: literal texta)</gloss>
<gloss>(lit: literal textb) gloss text3</gloss>
<gloss>gloss4 (lit: literal textc) more gloss</gloss>
<gloss>(lit: literal textd)</gloss>
<gloss>text8 (lit: with (parens) embedded) text2</gloss>
<gloss>text5a (lit: with (parens) embedded)(gloss parens) text5b</gloss>
<gloss>text6a (gloss parens)(lit: with (parens) embedded) text6b</gloss>
</sense>
</entry>

<!-- Lsource: text/no-text; eng/non-eng; wasei?; part? -->
<entry>
<!-- lsource eng -->
<ent_seq>3000090</ent_seq>
<r_ele>
<reb>あえ</reb>
</r_ele>
<sense>
<lsource>lsctext</lsource>
<gloss>test</gloss>
</sense>
</entry>


<entry>
<!-- lsource non-eng -->
<ent_seq>3000100</ent_seq>
<r_ele>
<reb>あえ</reb>
</r_ele>
<sense>
<lsource xml:lang="fre">lsctext</lsource>
<gloss>test</gloss>
</sense>
</entry>

<entry>
<!-- lsource wasei, text -->
<ent_seq>3000110</ent_seq>
<r_ele>
<reb>あえ</reb>
</r_ele>
<sense>
<lsource ls_wasei="y">lsctext</lsource>
<gloss>test</gloss>
</sense>
</entry>

<entry>
<!-- lsource eng, multi-word -->
<ent_seq>3000120</ent_seq>
<r_ele>
<reb>あえ</reb>
</r_ele>
<sense>
<lsource>multiword lsctext</lsource>
<gloss>test</gloss>
</sense>
</entry>

<entry>
<!-- lsource non-eng, multi-word, non-ascii -->
<ent_seq>3000130</ent_seq>
<r_ele>
<reb>あえ</reb>
</r_ele>
<sense>
<lsource xml:lang="fre">lsctext abcëăz</lsource>
<gloss>test</gloss>
</sense>
</entry>

