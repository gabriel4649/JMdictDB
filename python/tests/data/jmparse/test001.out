\set ON_ERROR_STOP 1
BEGIN;

COPY kwsrc(id,kw,descr,dt,notes,seq,sinc,smin,smax) FROM STDIN;
1	jmdict	\N	\N	\N	seq_jmdict	10	1000000	8999999
\.

COPY entr(id,src,stat,seq,dfrm,unap,srcnote,notes) FROM STDIN;
1	1	2	3000010	\N	False	\N	\N
2	1	2	3000020	\N	False	\N	\N
3	1	2	3000030	\N	False	\N	\N
4	1	2	3000040	\N	False	\N	\N
5	1	2	3000050	\N	False	\N	\N
6	1	2	3000060	\N	False	\N	\N
7	1	2	3000070	\N	False	\N	\N
8	1	2	3000080	\N	False	\N	\N
9	1	2	3000090	\N	False	\N	\N
10	1	2	3000100	\N	False	\N	\N
11	1	2	3000110	\N	False	\N	\N
12	1	2	3000120	\N	False	\N	\N
13	1	2	3000130	\N	False	\N	\N
\.

COPY kanj(entr,kanj,txt) FROM STDIN;
3	1	発音
5	1	発音
\.

COPY rdng(entr,rdng,txt) FROM STDIN;
1	1	あああ
2	1	テスト
3	1	はつおん
4	1	はつおん
4	2	ばしょう
4	3	バショー
5	1	はつおん
6	1	あい
7	1	あえ
8	1	あえ
9	1	あえ
10	1	あえ
11	1	あえ
12	1	あえ
13	1	あえ
\.

COPY sens(entr,sens,notes) FROM STDIN;
1	1	\N
2	1	\N
3	1	\N
4	1	\N
5	1	\N
6	1	\N
6	2	\N
6	3	\N
7	1	\N
8	1	\N
9	1	\N
10	1	\N
11	1	\N
12	1	\N
13	1	\N
\.

COPY gloss(entr,sens,gloss,lang,ginf,txt) FROM STDIN;
1	1	1	1	1	test10
2	1	1	1	1	test20
3	1	1	1	1	test30
4	1	1	1	1	test40
5	1	1	1	1	test50
6	1	1	1	1	test60-1
6	2	1	1	1	test60-2
6	3	1	1	1	test60-3
7	1	1	1	1	test70-1
7	1	2	1	1	test70-2
7	1	3	1	1	test70-3
8	1	1	1	1	gloss text2 (lit: literal texta)
8	1	2	1	1	(lit: literal textb) gloss text3
8	1	3	1	1	gloss4 (lit: literal textc) more gloss
8	1	4	1	1	(lit: literal textd)
8	1	5	1	1	text8 (lit: with (parens) embedded) text2
8	1	6	1	1	text5a (lit: with (parens) embedded)(gloss parens) text5b
8	1	7	1	1	text6a (gloss parens)(lit: with (parens) embedded) text6b
9	1	1	1	1	test
10	1	1	1	1	test
11	1	1	1	1	test
12	1	1	1	1	test
13	1	1	1	1	test
\.

COPY lsrc(entr,sens,ord,lang,txt,part,wasei) FROM STDIN;
9	1	1	1	lsctext	False	False
10	1	1	137	lsctext	False	False
11	1	1	1	lsctext	False	True
12	1	1	1	multiword lsctext	False	False
13	1	1	137	lsctext abcëăz	False	False
\.

COMMIT
